modules/todoitems.js

```diff
@@ -0,0 +1,166 @@
+import select2 from '../img/select.png';
+import delete2 from '../img/delete.png';
+import edit from '../img/edit.png';
+import save from '../img/save.png';
+
+export default class toDoItems {
+  constructor(text, completed, index) {
+    this.text = text;
+    this.completed = completed;
+    this.index = index;
+  }
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222570017-bf966af3-5590-4a6e-a585-4dbbb6e3a187.png)

- [ ] Kindly remove this Live Demo Section since you don't have a live link so it would be nice if you remove this section to make your readme file descriptive and professional 😊 Once you would have your live link you can add this section again with the working live link 👍